timemachine/fe/mle.py

```diff
@@ -212,3 +215,60 @@ def infer_node_vals_and_errs(
     dg_err = bootstrap_estimates.std(0)
 
     return dg, dg_err
+
+
+def infer_node_vals_and_errs_networkx(
+    nx_graph: nx.Graph,
+    edge_diff_prop: str,
+    edge_stddev_prop: str,
+    ref_nodes: Sequence[Any],
+    ref_node_val_prop: str,
+    ref_node_stddev_prop: Optional[str] = None,
+    n_bootstrap: int = 100,
+    seed: int = 0,
+) -> Tuple[NDArray, NDArray]:
+    """Version of :py:func:`fe.mle.infer_node_vals_and_errs` that accepts a networkx graph.
```

Comment:
Be explicit in the parent namespace of this function

```suggestion
    """Version of :py:func:`timemachine.fe.mle.infer_node_vals_and_errs` that accepts a networkx graph.
```